services/ui-src/src/components/modals/AddEditProgramModal.tsx

```diff
@@ -18,6 +18,7 @@ import { mcparReportJsonNested } from "forms/mcpar";
 export const AddEditProgramModal = ({
   activeState,
   selectedReportId,
+  selectedReportData,
```

Comment:
Because we already have the concept of `reportData`, maybe we could name all the `reportData` stuff in this PR to `reportMetadata` or something? What does everyone think?